federation/pkg/dnsprovider/providers/coredns/rrsets.go

```diff
@@ -78,13 +77,10 @@ func (rrsets ResourceRecordSets) Get(name string) (dnsprovider.ResourceRecordSet
 		}
```

Comment:
@miekg thanks for confirming that. Does it make sense to add a helper to `middleware/etc/msg` package to determine the record type instead of having to re-implement it in all the clients?

@shashidharatd thanks! Please let me know when you add it.